src/automaton/core/interop/ethereum/eth_helper_functions.h

```diff
@@ -34,7 +36,7 @@ static size_t curl_callback(void *contents, size_t size, size_t nmemb, std::stri
     s->append(reinterpret_cast<char*>(contents), new_length);
   }
   catch (std::bad_alloc &e) {
-    LOG(ERROR) << "Bad_alloc while reading data!";
+    LOG(WARNING) << "Bad_alloc while reading data!";
```

Comment:
We should revert all the ERROR -> WARNING changes and do a #define if necessary.